package.json

```
@@ -1,103 +1,104 @@
-    "@types/lodash.flatmap": "^4.5.6",
-    "@types/multicodec": "^1.0.0",
-    "@types/node": "^14.14.16",
-    "@types/qs": "^6.9.5",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "@types/react-redux": "^7.1.14",
-    "@types/react-router-dom": "^5.1.6",
-    "@types/react-virtualized-auto-sizer": "^1.0.0",
-    "@types/react-window": "^1.8.2",
-    "@types/rebass": "^4.0.7",
-    "@types/styled-components": "^5.1.7",
-    "@uniswap/default-token-list": "^1.3.1",
-    "@uniswap/token-lists": "^1.0.0-beta.16",
-    "@uniswap/v2-core": "1.0.0",
-    "@uniswap/v2-periphery": "^1.1.0-beta.0",
-    "@web3-react/core": "^6.0.9",
-    "@web3-react/fortmatic-connector": "^6.0.9",
-    "@web3-react/injected-connector": "^6.0.7",
-    "@web3-react/portis-connector": "^6.0.9",
-    "@web3-react/walletconnect-connector": "^6.1.1",
-    "@web3-react/walletlink-connector": "^6.0.9",
-    "ajv": "^6.12.3",
-    "cids": "^1.0.0",
-    "copy-to-clipboard": "^3.2.0",
-    "ethers": "^5.0.7",
-    "i18next": "^15.0.9",
-    "i18next-browser-languagedetector": "^3.0.1",
-    "i18next-xhr-backend": "^2.0.1",
-    "inter-ui": "^3.13.1",
-    "jazzicon": "^1.5.0",
-    "lodash.flatmap": "^4.5.0",
-    "multicodec": "^2.0.0",
-    "multihashes": "^3.0.1",
-    "polished": "^3.3.2",
-    "react": "^17.0.1",
-    "react-device-detect": "^1.6.2",
-    "react-dom": "^17.0.1",
-    "react-feather": "^2.0.8",
-    "react-i18next": "^10.7.0",
-    "react-popper": "^2.2.3",
-    "react-redux": "^7.2.0",
-    "react-router-dom": "^5.0.0",
-    "react-scripts": "^4.0.1",
-    "react-spring": "^8.0.27",
-    "react-virtualized-auto-sizer": "^1.0.2",
-    "react-window": "^1.8.5",
-    "rebass": "^4.0.7",
-    "redux-localstorage-simple": "^2.3.1",
-    "styled-components": "^5.2.1",
-    "typescript": "^4.1.3"
-  },
-  "devDependencies": {
-    "@pancakeswap-libs/eslint-config-pancake": "1.0.0",
-    "@types/jest": "^26.0.20",
-    "@types/testing-library__cypress": "^5.0.8",
-    "cypress": "^4.11.0",
-    "eslint": "^7.2.0",
-    "prettier": "^2.1.2",
-    "qs": "^6.9.4",
-    "serve": "^11.3.0",
-    "start-server-and-test": "^1.11.0"
-  },
-  "resolutions": {
-    "@walletconnect/web3-provider": "1.1.1-alpha.0"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject",
-    "lint": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
-    "lint:fix": "eslint 'src/**/*.{js,jsx,ts,tsx}' --fix",
-    "integration-test": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run'"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "license": "GPL-3.0-or-later"
-}
+{
+  "name": "@pancakeswap/interface",

```

Maybe let's change na din sa `package.json` yung name and description? So it would look custom-built for SparkSwap talaga

```suggestion
  "name": "@sparkswap/interface",
  "description": "SparkSwap Interface",
```

---

sige I'll do that po

---

@harveyjavier please see latest commit kuya. Thanks!

---

slr @kojiadrianojr, kaka lants lang. approved!